src/components/Gui.svelte

```diff
@@ -1,78 +1,66 @@
 <script>
-    import { theme } from "../theme";
+  import { theme } from '../theme'
 </script>
 
-<div id="gui" style="background-color: {$theme.palette.base.background}">
-    <div id="gui-titlebar" class="flex flex-row">
-        <div class="flex-none">
-            <img src="images/titlebar-left.png" alt="" />
-        </div>
-        <div class="flex-grow" />
-        <div class="flex-none">
-            <img src="images/titlebar-right.png" alt="" />
-        </div>
-    </div>
-    <div id="gui-window" class="">
-        <div id="gui-menu" class="flex flex-row items-center">
```

Comment:
In general we want to use `class` and have no styling inline except things which might change.